src/fare-contracts/components/ScrollContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import {StyleSheet} from '@atb/theme';
+import {PropsWithChildren} from 'react';
+import {View} from 'react-native';
+import {RefreshControl, ScrollView} from 'react-native-gesture-handler';
+
+type Props = PropsWithChildren<{isRefreshing: boolean; refresh: () => void}>;
+
+export const ScrollContainer = ({children, isRefreshing, refresh}: Props) => {
+  const styles = useStyles();
+
+  return (
+    <View style={styles.container}>
+      <ScrollView
+        style={styles.scrollView}
+        refreshControl={
+          <RefreshControl refreshing={isRefreshing} onRefresh={refresh} />
+        }
+      >
+        {children}
+      </ScrollView>
+    </View>
+  );
+};
```

Comment:
I'm not a fan of adding these generic reusable components inside a specific module like "fare-contracts". The name `ScrollContainer` is so generic that when it pops up in autocomplete-suggestions it seems like a reusable component for all the screens.

I would suggest leaving this component out of this PR, and if you find that having this reusable compoent is valuable, then do it in its own PR where we add it in `src/components`. And in that PR replace `ScrollView`s thoughout the app with this new compoent, and add JSdoc about the purpose of the component and when to use it.

Reply:
I think since scroll is not needed due to FullScreenView already handling scroll, this can be removed. Thanks!